### PR TITLE
i18n(zh-cn): translate guides/integrations-guide/image.mdx

### PR DESCRIPTION
**Pull request by yiyuan9**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate guides/integrations-guide/image.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/image.mdx [hunk 1]
```diff
@@ -0,0 +1,730 @@
...
+<p>
+
+**类型：** `string`<br/>
+**必填：** `true`
+
+</p>
+
+HTMLImageElement 属性 `sizes` 允许你为一系列媒体条件指定图像的布局宽度。
+
+有关更多详细信息，请参见 [MDN](https://developer.mozilla.org/zh-cn/docs/Web/API/HTMLImageElement/sizes)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/image.mdx:349]:
```suggestion
有关更多详细信息，请参见 [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes)。
```

File: src/content/docs/zh-cn/guides/integrations-guide/image.mdx [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+
+> 这在默认的 Squoosh 服务中不受支持。有关使用 `sharp` 服务的详细信息，请参见[安装部分](#安装-sharp-可选)。详细了解[如何使用 `sharp` 调整图像大小](https://sharp.pixelplumbing.com/api-resize)。
+
+在 `fit` 属性为 `cover` 或 `contain` 时，裁剪的位置。
+
+### `<Picture />`
+
+内置的 `<Picture />` 组件用于创建一个优化的 `<picture />`，既可以用于通过URL访问的远程图像，也可以用于从项目的 `src/` 目录中导入的本地图像。
+
+除了组件特定的属性外，`<Picture />` 组件中包含的 `<img />` 的任何有效HTML属性也将包含在生成的 `<img />` 中。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/image.mdx:306]:
```suggestion
除了组件特定的属性外，`<Picture />` 组件中包含的 `<img />` 的任何有效 HTML 属性也将包含在生成的 `<img />` 中。
```

File: src/content/docs/zh-cn/guides/integrations-guide/image.mdx [hunk 3]
```diff
@@ -0,0 +1,730 @@
...
+
+## 配置
+
+集成可以配置为与不同的图像服务一起运行，可以是托管的图像服务，也可以是在构建或 SSR 部署中本地运行的完整图像转换器。
+
+> 在开发过程中，本地图像可能尚未发布，也不会对托管的图像服务可用。在使用 `astro dev` 时，本地图像将始终使用内置的图像服务。
+
+### config.serviceEntryPoint
+
+`serviceEntryPoint` 应该解析为从NPM安装的图像服务。默认的入口点是 `@astrojs/image/squoosh`，它会解析为从此集成的 `package.json` 导出的入口点。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/image.mdx:481]:
```suggestion
`serviceEntryPoint` 应该解析为从 NPM 安装的图像服务。默认的入口点是 `@astrojs/image/squoosh`，它会解析为从此集成的 `package.json` 导出的入口点。
```

File: src/content/docs/zh-cn/guides/integrations-guide/image.mdx [hunk 4]
```diff
@@ -0,0 +1,730 @@
...
+// 裁剪为特定的宽度和高度
+<Image src={imageUrl} width={750} height={250} format="avif" alt="descriptive text" />
+
+// 高度将被重新计算以匹配宽高比
+<Image src={imageUrl} width={750} aspectRatio={16 / 9} format="avif" alt="descriptive text" />
+```
+
+### 响应式图片
+
+`<Picture />` 组件可用于自动构建具有多个尺寸和格式的 `<picture>` 。查看 [MDN](https://developer.mozilla.org/zh-cn/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction) 以深入了解响应式图片和美术方向。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/image.mdx:612]:
```suggestion
`<Picture />` 组件可用于自动构建具有多个尺寸和格式的 `<picture>` 。查看 [MDN](https://developer.mozilla.org/zh-CN/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction) 以深入了解响应式图片和美术方向。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work peeps 🙌

**Pull request merged**